### DISCUSSION TITLE
Is the plugin supposed to work on the new iOS app?

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [javalent/obsidian-leaflet#158] [General]
Or am I doing something wrong? 

I can't even turn the plugin on in the app?

![IMG_3C1341C00BF8-1](https://user-images.githubusercontent.com/4631971/125279416-369c5c00-e314-11eb-8c34-738736a0d72d.jpeg)

**Comment by valentine195** [comment]
I have it set to desktop only for the time being. Some of the libraries I use probably will not work on mobile, and I'm not sure the UX would be very nice. Now that the public release is out I can start doing some testing.

**Comment by valentine195** [comment]
Forgot about this - mobile support was added in 3.20.0.

**Reply by oyvindsolstad** [reply]
Works like a charm!
